commitlint/plugins.ts

```diff
@@ -262,6 +262,44 @@ export abstract class Plugins {
         ];
     }
 
+    public static defaultRevertMessage(
+        headerStr: string,
+        bodyStr: string | null,
+        when: string
+    ) {
+        let offence = false;
+        let isRevertCommitMessage = headerStr.toLowerCase().includes("revert");
```

Comment:
@realmarv shouldn't it be "startsWith" instead of "includes"?

Reply:
If someone changes the commit title isn't it possible that "revert" word comes somewhere else in the title?

Reply:
Yes, in which would mean that the revert message used is not default, so if when=never, then offence should be false in this case.

Reply:
I added a test for this case

Reply:
cool, but why do I still see the use of includes()?

Reply:
Because this variable is checking if the commit message is a revert commit message not if it's a default revert message